tests/metrics/regression/test_ssim.py

```diff
@@ -50,7 +51,7 @@ def _sk_metric(preds, target, data_range, multichannel):
     [(i.preds, i.target, i.multichannel) for i in _inputs],
 )
 class TestSSIM(MetricTester):
-    atol = 1e-3  # TODO: ideally tests should pass with lower tolerance
+    atol = 6e-5
 
     # TODO: for some reason this test hangs with ddp=True
     # @pytest.mark.parametrize("ddp", [True, False])
```

Comment:
Do tests pass with ddp=True?

Reply:
The test hangs with ddp=True

Reply:
@ydcjeff I just tried locally and the test does not hang for me when `ddp=True`.

Reply:
When I tried before, the test hangs on drone, will try again.

Reply:
Okay test still hangs on drone, so lets just disable ddp. Really not sure why it is not working.